SECURITY.md

```diff
@@ -0,0 +1,33 @@
+Thanks for helping make GitHub safe for everyone ❤️
+
+## Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [desktop](https://github.com/desktop).
```

Comment:
Nit: the name of the org is `GitHub Desktop`, but feel free to ignore this change 🤣 
```suggestion
GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub Desktop](https://github.com/desktop).
```